molecule/common/playbook_template.yml

```diff
@@ -212,6 +225,18 @@
             server_name: localhost
             error_page: /usr/share/nginx/html
             autoindex: false
+            sub_filter:
+              sub_filters:
+                - 'server_hostname' '$hostname';
```

Comment:
Looks like tests are failing. The error is due to how you are defining the `sub_filters`. YAML doesn't like the four single quotes per line. (You should also remove the semicolon at the end of each item in the list since the `sub_filter` template will automatically add the semicolon to each `sub_filter).